src/Operation/Find.php

```diff
@@ -351,6 +367,27 @@ private function createQuery()
             $options['modifiers'] = $modifiers;
         }
 
-        return new Query($this->filter, $options);
+        return $options;
+    }
+
+    /**
+     * Create options for executing the command.
+     *
+     * @see http://php.net/manual/en/mongodb-driver-server.executequery.php
+     * @return array
+     */
+    private function createOptions()
```

Comment:
`createExecuteOptions()` might be clearer. I might suggest this for any files where we'll now have multiple functions creating "options". I realize the doc block explains this, but a rename will ensure it's perfectly clear what the method does from the places it is being called.